ci-operator/config/openshift/azure-disk-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml

```diff
@@ -3,11 +3,23 @@ base_images:
     name: "4.8"
     namespace: ocp
     tag: base
+  ocp_builder_rhel-8-golang-1.16-openshift-4.8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.8
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
     tag: golang-1.15
```

Comment:
Don't we need to change this too?

Reply:
I've updated this to `golang-1.16` for the master, 4.8, and 4.9 jobs.